sys/net/application_layer/dhcpv6/client.c

```diff
@@ -935,6 +1218,33 @@ static void _rebind(event_t *event)
     _request_renew_rebind(DHCPV6_REBIND);
 }
 
+static void _deprecate_remove_addrs(event_t *event)
+{
+    if (!IS_USED(MODULE_DHCPV6_CLIENT_IA_NA)) {
+        return;
+    }
+
+    (void)event;
+
+    for (unsigned i = 0; (i < CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX); i++) {
```

Comment:
and here

```suggestion
    for (int i = 0; (i < CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX); i++) {
```

Reply:
(and anywhere else where `CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX` is compared with an `unsigned`)